ui/jobs/jobs.js

```diff
@@ -3886,36 +3889,56 @@ class Bars {
     const log = e.rawLine;
 
     const type = line[0];
+    const trackedDoTs = Object.keys(this.mobGainEffectFromYouFuncMap);
```

Comment:
This is maybe premature optimization, but rather than calling this once per log line, can you set `this.trackedDots` once after the job setup?

Reply:
Oops, didn't catch this issue!